config/routes.rb

```diff
@@ -39,6 +39,7 @@
   get  'update_users'     => 'admin/users#update_users',      as: :update_users
   post 'update_status'    => 'admin/users#update_status',     as: :update_status
   resources :positions, controller: 'admin/positions', only: [ :create, :update, :destroy ]
+  resources :users, controller: 'admin/users', only: [ :edit, :update ]
```

Comment:
Space inside square brackets detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.